modules/api_loader.lua

```diff
@@ -51,8 +64,7 @@ local function LoadParamsInFunction(param, interface)
   return name, data
 end
 
-
-
+--- Load Enums values
```

Comment:
@pestOO is it doxygen comment? 
DO doxygen parse all comments? Or there is special marks or doxygen comments